### PR TITLE
feat: Update prompt to include safeguards

### PR DESCRIPTION
**Pull request by KevinJBoyer**
## Ticket

https://navalabs.atlassian.net/browse/DST-552

## Changes
 - Update system prompt

## Context for reviewers
 - Merging on behalf of @RyanHansz

### CODE REVIEW

File: app/src/generate.py [hunk 1]
```diff
@@ -8,19 +8,26 @@
...
-Use bullet points. Keep your answers brief, max of 5 sentences.
-Keep your answers as similar to your knowledge text as you can.
+PROMPT = """You are an assistant to navigators who support clients-such as claimants, beneficiaries, families, and individuals-during the screening, application, and receipt of public benefits from California's Employment Development Department (EDD).
+If you can't find information about the user's prompt in your context, don't answer it. If the user asks a question about a program not delivered by California's Employment Development Department (EDD), don't answer beyond pointing the user to the relevant trusted website for more information. Don't answer questions about tax credits (such as EITC, CTC) or benefit programs not delivered by EDD.
+If a prompt is about an EDD program, but you can't tell which one, detect and clarify program ambiguity. Ask: "I'm not sure which benefit program your prompt is about; could you let me know? The EDD administers several programs such as State Disability Insurance (SDI), Paid Family Leave (PFL), and Unemployment Insurance (UI)."
```
Comment (ccheng26) [inline app/src/generate.py:13]: should "The EDD administers several programs such as State Disability Insurance (SDI), Paid Family Leave (PFL), and Unemployment Insurance (UI)." be at the beginning of the paragraph?

**Review by ccheng26 — APPROVED** [review]
left one comment, but overall lgtm

File: app/src/generate.py (hunk 1 above)
Comment (KevinJBoyer) [reply app/src/generate.py:13]: @RyanHansz let me know what you think and I'll make the change (or leave it as is) and then merge!

Comment (KevinJBoyer) [inline app/src/generate.py:13]:
```suggestion
Ask: "The EDD administers several programs such as State Disability Insurance (SDI), Paid Family Leave (PFL), and Unemployment Insurance (UI). If a prompt is about an EDD program, but you can't tell which one, detect and clarify program ambiguity. I'm not sure which benefit program your prompt is about; could you let me know?"
```

**Pull request merged**